src/patternfly/components/Table/examples/Table.md

```diff
@@ -621,6 +621,92 @@ These classes can be used to ensure that the table changes between the tabular a
 {{/table}}
 ```
 
+### Table with buttons and actions
+```hbs
+{{#> table table--id="table-buttons-and-actions" table--IsGrid=true table--modifier="pf-m-grid-md" table--attribute='aria-label="This is a table with buttons and actions"'}}
+  {{#> table-thead}}
+    {{#> table-tr}}
+      {{#> table-th table-th--attribute='scope="col"'}}
+        Deployment
+      {{/table-th}}
+      {{#> table-th table-th--attribute='scope="col"'}}
+        Status
+      {{/table-th}}
+      {{#> table-th table-th--attribute='scope="col"'}}
+        Builds
+      {{/table-th}}
+      {{#> table-th table-th--attribute='scope="col"'}}
+        Start New Build
+      {{/table-th}}
+      {{> table-cell-action}}
+    {{/table-tr}}
+  {{/table-thead}}
+
+  {{#> table-tbody}}
+    {{#> table-tr}}
+      {{#> table-td table-td--data-label="Deployment"}}
+        Deployment 1
+      {{/table-td}}
+      {{#> table-td table-td--data-label="Status"}}
+        Success
+      {{/table-td}}
+      {{#> table-td table-td--data-label="Builds"}}
+        6
+      {{/table-td}}
+      {{#> table-td table-td--data-label="Start Build" table-td--modifier="pf-m-action"}}
+        <span class="pf-v6-c-table__text">
```

Comment:
Can you make this a regular `<span>`? the `__text` element works, but it's more intended for use with the modifiers that manipulate text (nowrap, truncate, etc)

```suggestion
        <span>
```